loader/db.go

```diff
@@ -131,6 +134,13 @@ func (conn *Conn) executeSQLCustomRetry(ctx *tcontext.Context, sqls []string, en
 
 		startTime := time.Now()
 		err = executeSQLImp(ctx, conn.db, sqls)
+
+		failpoint.Inject("LoadExecCreateTableFailed", func() {
+			if i == 0 && len(sqls) == 1 && strings.Contains(sqls[0], "CREATE TABLE") {
+				err = domain.ErrInfoSchemaChanged
```

Comment:
why only inject `ErrInfoSchemaChanged ` error here? can we support to inject custom error?

Reply:
use error code, https://github.com/pingcap/dm/pull/206/commits/3b3579559aa458333d25826569faea6fcd36d828